scripts/Text Grammar Corrector/grammer bot.py

```diff
@@ -0,0 +1,57 @@
+# Imports
+import language_tool_python
+import argparse
+from pathlib import Path
+
+# ------------------------------------------------------
+# Initialize API
+tool = language_tool_python.LanguageToolPublicAPI("en-US")
+
+# get the input text file from the user
+parser = argparse.ArgumentParser()
+parser.add_argument("input_file_path", type=Path)
+p = parser.parse_args()
```

Comment:
Nit pick. Can you add a -h flag for usage/help and make the file path argument mandatory for the script to run?

Reply:
Done